sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Azure.Monitor.OpenTelemetry.Exporter.csproj

```diff
@@ -1,13 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>An OpenTelemetry .NET exporter that exports to Azure Monitor</Description>
     <AssemblyTitle>AzureMonitor OpenTelemetry Exporter</AssemblyTitle>
     <Version>1.1.0-beta.1</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
     <ApiCompatVersion>1.0.0</ApiCompatVersion>
     <PackageTags>Azure Monitor OpenTelemetry Exporter ApplicationInsights</PackageTags>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net6.0</TargetFrameworks>
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
+
+    <!--TODO: ApiCompat is failing because I added net6. Will follow up how to correctly resolve.-->
+    <ApiCompatBaselineTargetFramework Condition="'$(TargetFramework)' == 'net6.0'">netstandard2.0</ApiCompatBaselineTargetFramework>
```

Comment:
I copied this from Azure.Core. Need to get confirmation that this is an appropriate fix for ApiCompat.


https://github.com/Azure/azure-sdk-for-net/blob/42f7757cf8f34d2743f55f28d7e776607ef174e8/sdk/core/Azure.Core/src/Azure.Core.csproj#L20C5-L20C125